rust/core-lib/src/tabs.rs

```diff
@@ -685,28 +681,36 @@ impl Documents {
     }
 
     /// Attempt to load a config file.
-    fn load_file_based_config(&mut self, path: &Path) {
+    fn load_file_based_config(&mut self, peer: &MainPeer, path: &Path) {
         match config::try_load_from_file(&path) {
-            Ok((d, t)) => self.set_config(d, t, Some(path.to_owned())),
-            Err(e) => eprintln!("Error loading config file: {:?}", e),
+            Ok((d, t)) => self.set_config(peer, d, t, Some(path.to_owned())),
+            Err(e) => {
+                let err_msg = format!("{}", &e);
+                peer.send_rpc_notification("alert", &json!({"msg": err_msg}));
+            }
         }
     }
 
     /// Sets (overwriting) the config for a given domain. Will fail if the config
     /// fails validation.
```

Comment:
Is "fails validation" still the most accurate text for this, or should it be slightly more general to cover other potential failures?